src/pybind/mgr/volumes/module.py

```diff
@@ -497,6 +502,7 @@ def __init__(self, *args, **kwargs):
         # for mypy
         self.max_concurrent_clones = None
         self.snapshot_clone_delay = None
+        self.clone_delay_every_regfile = None
```

Comment:
Copy operation is too fast to detect whether or not the progress bar was updated.

time required for copying a file is much much smaller compared to "time required by mgr/vol to pass update to mgr/progress + time required to update progress bar by mgr/progress module + time required by Python QA code to detect the update".

Reply:
> Yes, it is required. Copy operation is too fast to detect whether or not the progress bar was updated.
> 
> time required for copying a file is much much smaller compared to "time required by mgr/vol to pass update to mgr/progress + time required to update progress bar by mgr/progress module + time required by Python QA code to detect the update".

Have you tried using a higher clone count (maybe also decrease the number of cloner threads) and/or bigger file sizes (10GB+) rather than 3 clones of 1GB?